app/assets/stylesheets/reports.scss

```diff
@@ -1,3 +1,21 @@
 // Place all the styles related to the Reports controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+$dcaf-color: #5D3E8D;
+
+.report-section{
+  margin-bottom: 50px;
+}
+
+.line-header, .line-header:hover{
+  color: $dcaf-color;
+}
+
+#budget_bar{
```

Comment:
If we can, let's drop this CSS and just make sure we apply a global style to the budget bar when it's done, since it's a separate feature. I think just the text placeholder is fine for now. Thoughts?